ui/src/main/resources/AzureAD/WebPreferences.xml

```diff
@@ -386,7 +387,7 @@
   #set ($value = $xwiki.getDefaultDocumentSyntax())
 #end
 &lt;select name="${object.getxWikiClass().name}_${object.number}_${name}" id="${object.getxWikiClass().name}_${object.number}_${name}"&gt;
-#set ($configuredSyntaxes = $services.rendering.getConfiguredSyntaxes())
+#set ($configuredSyntaxes = $collectiontool.sort($services.rendering.getConfiguredSyntaxes()))
```

Comment:
Why is needed to sort them?

Reply:
Was modified on export from XWiki instance, version 14.10.